src/deepsparse/transformers/server/throttled_engine.py

```diff
@@ -94,12 +94,15 @@ def get_throttled_engine_pipeline(
         deepsparse.transformers module
     """
     _pipeline_config: PipelineEngineConfig = config or PipelineEngineConfig.get_config()
+    if _pipeline_config.task is None:
+        return None
     _pipeline_engine: Pipeline = pipeline(
```

Comment:
nit: variable name inside function doesn't have a reason to start with `_` here

Reply:
Addressed!